rblib/format.rb

```diff
@@ -81,8 +81,8 @@ def self.make_clickable(text, params = {})
         # get &gt; contaminated into the linked URL)
         def self.simplify_angle_bracketed_urls(text)
             ret = ' ' + text + ' '
-            #ret = ret.gsub(/(www\.[^\s<>{}()])\s+\<(https?):\/\//i, "\\1")
-            ret = ret.gsub(/(www\.[^\s<>{}()]+)\s+<http:\/\/\1>/i, "\\1")
+            #ret = ret.gsub(/(www\.[^\s<>])\s+\<(https?):\/\//i, "\\1")
+            ret = ret.gsub(/(www\.[^\s<>]+)\s+<http:\/\/\1>/i, "\\1")
```

Comment:
Could the commented line be removed (doesn't seem to be for anything, or work), and the second line cope with https too?